### DISCUSSION TITLE
How to get profile twitter username into session?

### DISCUSSION THREAD

**Discussion by xpluscal** [nextauthjs/next-auth#8863] [Help]
### Summary

Using auth/core
and auth/sveltekit.

The callbacks in the config, aka jwt and session both do not contain the profile data in order to add it.
Adding it to a customized profile function doesn't change that return either.

Is there another way to make sure the username lands in the jwt token so it's available in session?

**Comment by SahilKharya** [comment]
Would like if someone has a solution to this
